translations/messages+intl-icu.de.yml

```diff
@@ -3742,7 +3742,6 @@ tag.name.empty.error: "Das Schlagwort muss einen Namen haben."
 tag.topic.name.empty.error: "Das Thema muss einen Namen haben."
 tag.topic.name.duplicate.error: "Das Thema existiert bereits. Bitte verwenden Sie einen anderen Namen."
 tag.new: "Neues Schlagwort"
```

Comment:
shouldn't we refactor this and several other keys to `entity.create` too? Maybe that would be a good onboarding-task :thinking: